### PR TITLE
feat(numericinput): update contract and tidy style

### PR DESCRIPTION
**Pull request by congcongcong250**
This change is [Reviewable](https://reviewable.io/reviews/qantasairways/runway/179)

### CODE REVIEW

**Comment by congcongcong250** [comment]
- Tidy global style without breaking
- Explicitly passing props into rc-numeric-input
- Unit test for accessibility

File: src/components/NumericInput/index.js [hunk 1]
```diff
@@ -301,7 +291,9 @@ NumericInput.defaultProps = {
   label: '',
   ariaDescription: '',
   onChange: () => {},
-  value: null
+  value: null,
+  min: undefined,
+  max: undefined
```
Comment (jamesb3ll) [inline src/components/NumericInput/index.js]: I'm not a big fan of eslint forcing to add defaultProps? There's not one useful defaultProp for this component. Could we remove this rule? What do you think? @tysonwolker @congcongcong250 @rachmulvey

Comment (rachmulvey) [reply src/components/NumericInput/index.js]: I'd be ok with that!

File: src/components/NumericInput/index.js [hunk 2]
```diff
@@ -121,12 +121,6 @@ const getRcInputNumberStyles = ({ highlightInvalid }) => ({
     overflow: 'hidden',
     height: '100%'
   },
-  [forAll(
-    SELECTORS.RCI.CONTROLS.DOWN.DISABLED,
-    SELECTORS.RCI.CONTROLS.UP.DISABLED
-  )]: {
-    opacity: '0.72'
-  },
```
Comment (congcongcong250) [inline src/components/NumericInput/index.js]: Notes: we don't use this rule at all

File: src/components/NumericInput/index.js [hunk 3]
```diff
@@ -163,7 +157,6 @@ const axValidationContainerStyles = {
 };
 class NumericInput extends Component {
   state = {
-    value: this.props.value,
```
Comment (congcongcong250) [inline src/components/NumericInput/index.js]: Notes: we don't use this rule at all

**Pull request closed without merging**